draft-ietf-quic-transport.md

```diff
@@ -5495,6 +5495,15 @@ Reason Phrase:
   This SHOULD be a UTF-8 encoded string {{!RFC3629}}.
 
 
+## HANDSHAKE_DONE frame {#handshake-done-frame}
+
+The server uses the HANDSHAKE_DONE frame (type=0x1e) to signal confirmation of
+the handshake to the client.  The HANDSHAKE_DONE frame contains no additional
+fields.
+
+This frame MUST be retransmitted until it is acknowledged by the client.
```

Comment:
You should move this statement to [Section 13.3](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#retransmission-of-information) for consistency reasons.